### PR TITLE
Add typings for @tonic-ui/react 1.10

### PR DESCRIPTION
**Pull request by derekhawker**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## tonic-ui__react ([<kbd>unpkg</kbd>](https://unpkg.com/browse/tonic-ui__react@latest/))
was missing the following properties:
1. AlertCloseButton
1. AlertIcon
1. AlertMessage
1. CSSBaseline
1. ColorModeProvider

<details>
<summary>as well as these 19 other properties...</summary>
<p>ColorStyleProvider, DrawerContainer, FlatButton, LinearProgress, ModalContainer, Popper, PopperArrow, PortalManager, PseudoBox, TagCloseButton, ThemeProvider, ToastCloseButton, ToastContainer, ToastIcon, ToastMessage, ToggleSwitch, useDisclosure, usePortalManager, withTheme</p>
</details>

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 59a2d6c354dd765b271809cae8e449ce71a9042e
</p>

**Comment by derekhawker** [comment]
Reviewed the list of missing properties. I added additional typings for some of the mentioned properties. Purposely leaving out the following:

**Deprecated since release of v1 and to be removed in next major version** (FlatButton, PseudoBox, ToggleSwitch, useDisclosure, withTheme)
**Actually an internal component and not meant to be used** (CSSBaseline, ColorModeProvider, ColorStyleProvider, ThemeProvider, AlertIcon,AlertMessage, DrawerContainer, ModalContainer, Popper, PopperArrow, ToastContainer, ToastIcon, ToastMessage)

File: types/tonic-ui__react/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,932 @@
+// Type definitions for @tonic-ui/react 1.10
+// Project: https://github.com/trendmicro-frontend/tonic-ui/tree/master/packages/react
+// Definitions by: Derek Hawker <https://github.com/derekhawker>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 3.7
```
Comment (sandersn) [inline types/tonic-ui__react/index.d.ts:5]: this isn't needed since the current minimum version is 4.3

**Review by sandersn — CHANGES_REQUESTED** [review]

**Review by sandersn — APPROVED** [review]

**Comment by derekhawker** [comment]
Ready to merge

**Pull request merged**